yt_dlp/extractor/megatvcom.py

```diff
@@ -0,0 +1,177 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+from urllib.parse import (
+    parse_qs,
+    urlparse,
+)
+
+from .common import InfoExtractor
+from ..utils import (
+    clean_html,
+    determine_ext,
+    ExtractorError,
+    extract_attributes,
+    get_element_by_class,
+    get_element_html_by_id,
+    HEADRequest,
+    unescapeHTML,
+    unified_timestamp,
+)
+
+
+class MegaTVComBaseIE(InfoExtractor):
+    _PLAYER_DIV_ID = 'player_div_id'
+
+    def _extract_player_attrs(self, webpage):
+        player_el = get_element_html_by_id(self._PLAYER_DIV_ID, webpage)
+        return {
+            re.sub(r'^data-(?:kwik_)?', '', k): v
+            for k, v in extract_attributes(player_el).items()
+            if k not in ('id',)
+        }
```

Comment:
It does lookup `self._PLAYER_DIV_ID`. I suppose I can just hard-code that or have a module-level constant. However my gut feeling is this might have to be different per extractor, even though that is not the case now, so I would rather keep it this way. If you insist I can change it though.